src/EPR.PRN.Backend.API/Controllers/PrnController.cs

```diff
@@ -204,5 +205,25 @@ public async Task<IActionResult> CalculateAsync(Guid organisationId, [FromBody]
         }
     }
 
+    [HttpPost("updatesyncstatus")]
```

Comment:
This method is actually doing only insert, but the method name is updatesyncstatus?